lib/main.js

```diff
@@ -25,6 +28,19 @@ let showSource;
 let disableExecuteTimeout;
 let excludedSniffs;
 
+const translateEncodingName = (name) => {
+  const canonical = iconvLiteAliases[name] || name;
+  let ret;
+  if (iconvTranslation[canonical]) {
+    ret = iconvTranslation[canonical];
```

Comment:
Yea, I remember the first time I hit that I was rather confused until I realized what it meant.